src/physics/burgers.cpp

```diff
@@ -212,15 +212,43 @@ ::source_term (
 
     using TestType = Parameters::AllParameters::TestType;
 
-    if(this->test_type == TestType::burgers_energy_stability){
+    if(this->test_type == TestType::burgers_energy_stability || this->test_type == TestType::burgers_limiter){
         for(int istate =0; istate<nstate; istate++){
             source[istate] = 0.0;
             const double pi = atan(1)*4.0;
-            for(int idim=0; idim< dim; idim++){
-              // source[istate] += pi * cos(pi*(pos[idim] - current_time))
-              //                     *(-0.99 + sin(pi * (pos[idim] - current_time)));
-               source[istate] += pi * sin(pi*(pos[idim] - current_time))
-                                   *(1.0 - cos(pi * (pos[idim] - current_time)));
+            real pi_xmt = pi * (pos[0] - current_time);
+            real pi_ymt = pi * (pos[1] - current_time);
+            real pi_zmt = pi * (pos[2] - current_time);
```

Comment:
always use `const` if variable are not meant to change after their declaration

Reply:
Changed these variables to const! 😄